### PR TITLE
test: Improve unit test coverage of nodeserver.go

### LINKED ISSUE #625: improve unit test coverage of nodeserver.go
**Is your feature request related to a problem?/Why is this needed**

**Describe the solution you'd like in detail**

Should add more unit tests for `NodeExpandVolume` func:
https://coveralls.io/builds/35694161/source?filename=pkg/azuredisk/nodeserver.go

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by kassarl**
**What type of PR is this?**
/kind test

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #625 

**Requirements**:
- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
- [ ] includes documentation
- [x] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

**Comment by kassarl** [comment]
I'm going to close this PR and investigate why its failing the MacOS build and then reopen when I find the solution

**Comment by andyzhangx** [comment]
you could just check error msg in macos build and send additional commit to fix it in this PR

File: pkg/azuredisk/nodeserver_test.go [hunk 1]
```diff
@@ -698,13 +702,54 @@ func TestNodeExpandVolume(t *testing.T) {
 				DefaultError: status.Error(codes.InvalidArgument, "volume path must be provided"),
 			},
 		},
+		{
+			desc: "Volume capacity invalid",
+			req: csi.NodeExpandVolumeRequest{
+				CapacityRange: stdCapacityRange,
```
Comment (andyzhangx) [inline pkg/azuredisk/nodeserver_test.go:708]: you could use visual studio code or `gofmt -s -w filename`  to format the code

**Comment by andyzhangx** [comment]
can you squash all commits to one `git rebase -i HEAD~4` and then force push?

**Review by andyzhangx — APPROVED** [review]
/lgtm
/approve
/retest

**Pull request merged**